index.html

```diff
@@ -3330,7 +3330,7 @@ <h3 class="no-num no-ref heading settled" id="index-defined-elsewhere"><span cla
      <li><a href="https://w3c.github.io/page-visibility#dfn-steps-to-determine-the-visibility-state">steps to determine the visibility state</a>
     </ul>
    <li>
-    <a data-link-type="biblio">[PERMISSIONS]</a> defines the following terms:
+    <a data-link-type="biblio">[permissions-1]</a> defines the following terms:
```

Comment:
When I process bikeshed thru `https://api.csswg.org/bikeshed/` I don't get permissions-1 or secure-contexts-1 links.

Reply:
@rwaldron, happy to see Bocoup join W3C!

To fix the nit, try updating your locally installed bikeshed per https://tabatkins.github.io/bikeshed/#updating-bikeshed or alternatively, build it using the HTTP API per https://tabatkins.github.io/bikeshed/#curl

This will make sure the spec is built with the most recent biblio database.

(@alexshalamov, feel free to merge when the nits have been addressed.)